src/benchmarks/real-world/ImageSharp/Codecs/Jpeg/DecodeJpeg_ImageSpecific.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using SixLabors.ImageSharp.Formats.Jpeg;
+using SixLabors.ImageSharp.Tests;
+
+// ReSharper disable InconsistentNaming
+namespace SixLabors.ImageSharp.Benchmarks.Codecs.Jpeg
+{
+    /// <summary>
+    /// Image-specific Jpeg benchmarks
+    /// </summary>
+    
+    public class DecodeJpeg_ImageSpecific
+    {
```

Comment:
@kunalspathak any chance you can update this as discussed above?

Reply:
Done.

Reply:
It's not pushed yet?